pegasus/sites.v3/code.org/views/dance_tutorial_section.haml

```diff
@@ -0,0 +1,22 @@
+.dance-subheading
```

Comment:
this feels a little more related to the overall dance page than to this partial - especially since it won't be included if you extract this to reuse, so I might move it to dance.haml

Reply:
Agree!  Thanks.